.travis.yml

```diff
@@ -9,6 +9,7 @@ julia:
   - 1.0
   - 1.3
   - 1.4
+  - 1.5
```

Comment:
Let's only test the latest stable release:
```suggestion
  - 1
```

Reply:
done in #295 which is merged here